### PR TITLE
Add dropdown button

### PR DESCRIPTION
**Pull request by tomquinonero**
This PR introduce a dropdown button feature.
It uses the actual dropdown implementation and uses a CSS enhanced button group.

Low  urgence on this one, we can take time.

### CODE REVIEW

File: packages/kotti-dropdown-button/KtDropdownButton.vue [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+				<KtButton
+					class="kt-dropdown-button__icon"
+					v-bind="$attrs"
+					:icon="dropdownIcon"
+				>
+					<i class="yoco" v-text="dropdownIcon" />
+				</KtButton>
+			</KtButtonGroup>
+			<div slot="content">
+				<slot name="KtDropdownMenus">Dropdown Menu</slot>
```
Comment (amrdraz) [inline packages/kotti-dropdown-button/KtDropdownButton.vue]: Why name the slot, why not just have it be the default slot

File: packages/kotti-dropdown-button/KtDropdownButton.vue [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+import KtButton from '../kotti-button'
+import KtButtonGroup from '../kotti-button-group'
+import KtPopover from '../kotti-popover'
+
+export default {
+	name: 'KtDropdownButton',
+	props: {
+		dropdownIcon: {
+			type: String,
+			default: 'chevron_up',
```
Comment (amrdraz) [inline packages/kotti-dropdown-button/KtDropdownButton.vue]: I think most dropdown default to chevrin_down not up
also opening the drop down down
it's very strange to see the arrow pointing up
the only exception is if the dropdown is in the bottom of the ui at which point maybe like popover we could have the icon default to poointing at the side the popover would open, unless it was explicitly set

File: packages/kotti-dropdown-button/KtDropdownButton.vue (hunk 1 above)
Comment (tomquinonero) [reply packages/kotti-dropdown-button/KtDropdownButton.vue]: I copied the behavior of the "classic" Dropdown. I don't think we should change it here if we don't change it in the `KtDoropdown` component as well.

File: packages/kotti-dropdown-button/KtDropdownButton.vue (hunk 2 above)
Comment (tomquinonero) [reply packages/kotti-dropdown-button/KtDropdownButton.vue]: Yes I kind of did that thinking of bottom buttons. I don't mind defaulting down.

File: docs/pages/components/dropdowns.vue [hunk 3]
```diff
@@ -31,6 +31,33 @@ In `KtDropdownMenus` slot, you can write your own stylied element. However, we s
 </KtDropdown>
 ```
 
+## Dropdown button 
+
+A dropdown button is a current pattern nowadays.
```
Comment (FlorianWendelborn) [inline docs/pages/components/dropdowns.vue]: I think this sentence should be removed.

File: packages/kotti-dropdown-button/KtDropdownButton.vue (hunk 1 above)
Comment (FlorianWendelborn) [reply packages/kotti-dropdown-button/KtDropdownButton.vue]: Slot name is very redundant. Just call it `menu` IMO.

File: docs/pages/components/dropdowns.vue [hunk 4]
```diff
@@ -31,6 +31,33 @@ In `KtDropdownMenus` slot, you can write your own stylied element. However, we s
...
+			<KtDropdownMenu>Download Item</KtDropdownMenu>
+		</div>
+	</KtDropdownButton>
+</div>
+
+``` html
+<div class="element-example">
+	<KtDropdownButton size="sm" type="primary" v-t="'Dropdown button'" >
+		<div slot="KtDropdownMenus">
+			<KtDropdownMenu>Request item</KtDropdownMenu>
```
Comment (FlorianWendelborn) [inline docs/pages/components/dropdowns.vue]: Looks more like a `KtDropdownItem` or `KtDropdownMenuItem` to me.

Comment (tomquinonero) [reply docs/pages/components/dropdowns.vue]: That is using the existing naming for KtDropdown :/

**Pull request merged**